components/app/interface/settings/client/index.tsx

```diff
@@ -196,6 +197,14 @@ const Config: SettingsConfiguration<{ server: Server }> = {
           {
             id: "logout",
             icon: <BiSolidExit size={20} color={theme!.colours.error} />,
+            href: "/login",
+            onClick: () => {
+              const acli = getController("state");
```

Comment:
This package should also have direct access to state and it can be called through `state.`